scripts/ccf/sandbox_local/join_config.json

```diff
@@ -0,0 +1,70 @@
+{
+    "enclave": {
+      "file": "/opt/ccf_virtual/lib/libjs_generic.virtual.so",
+      "type": "Release",
+      "platform": "Virtual"
+    },
+    "network": {
+      "node_to_node_interface": {
+        "bind_address": "127.0.0.1:0",
+        "published_address": ""
+      },
+      "rpc_interfaces": {
+        "primary_rpc_interface": {
+          "app_protocol": "HTTP2",
+          "bind_address": "127.0.0.1:0",
+          "endorsement": {
+            "authority": "Service"
+          },
+          "forwarding_timeout_ms": 3000,
+          "http_configuration": {
+            "initial_window_size": "65536",
+            "max_body_size": "1048576",
+            "max_concurrent_streams_count": 100,
+            "max_frame_size": "16384",
+            "max_header_size": "16384",
+            "max_headers_count": 256
+          },
+          "max_open_sessions_hard": 1010,
+          "max_open_sessions_soft": 1000,
+          "protocol": "tcp",
+          "published_address": "127.0.0.1:0"
+        },
+        "secondary_rpc_interface": {
+          "app_protocol": "HTTP2",
+          "bind_address": "127.0.0.1:0",
+          "endorsement": {
+            "authority": "Node"
+          },
+          "forwarding_timeout_ms": 3000,
+          "http_configuration": {
+            "initial_window_size": "65536",
+            "max_body_size": "1048576",
+            "max_concurrent_streams_count": 100,
+            "max_frame_size": "16384",
+            "max_header_size": "16384",
+            "max_headers_count": 256
+          },
+          "max_open_sessions_hard": 1010,
+          "max_open_sessions_soft": 1000,
+          "protocol": "tcp",
+          "published_address": "127.0.0.1:0"
+        }
+      }
+    },
+    "command": {
+      "type": "Join",
+      "service_certificate_file": "/workspace/sandbox_common/service_cert.pem",
+      "join": {
+        "retry_timeout": "1s",
+        "target_rpc_address": "127.0.0.1:8000",
+        "follow_redirect": true
+      }
+    },
+    "output_files": {
+      "node_certificate_file": "${NODE_WORKSPACE}/0.pem",
```

Comment:
are we using the same file structure as sandbox?

Reply:
Yes where it matters
```
workspace/
├── join_config.json
├── node-b621a4a8-8fa7-40dc-8451-83ee87644753
│   ├── 0.node_address
│   ├── 0.pem
│   ├── 0.pid
│   ├── 0.rpc_addresses
│   ├── err
│   └── out
├── proposals
│   └── trust_0b374cca774ad0322ef0141fdbf07cdd66b59614b6bb1281da2651ee98a43cce.json
├── sandbox_0
│   ├── 0.config.json -> /workspace/sandbox_common/0.config.json
│   ├── 0.ledger
│   │   ├── ledger_1-10.committed
│   │   └── ledger_11
│   ├── 0.node_address
│   ├── 0.pem
│   ├── 0.rpc_addresses
│   ├── 0.snapshots
│   │   └── snapshot_10_11.committed
│   ├── actions.js
│   ├── apply.js
│   ├── cchost -> /opt/ccf_virtual/bin/cchost
│   ├── err
│   ├── libjs_generic.virtual.so -> /opt/ccf_virtual/lib/libjs_generic.virtual.so
│   ├── member0_cert.pem
│   ├── member0_enc_pubk.pem
│   ├── node.pid
│   ├── out
│   ├── resolve.js
│   ├── service_cert.pem
│   └── validate.js
└── sandbox_common
    ├── 0.config.json
    ├── 0.node_address
    ├── 0.pem
    ├── 0.rpc_addresses
    ├── actions.js
    ├── apply.js
    ├── keygenerator.sh -> /opt/ccf_virtual/bin/keygenerator.sh
    ├── member0_cert.pem
    ├── member0_enc_privk.pem
    ├── member0_enc_pubk.pem
    ├── member0_privk.pem
    ├── resolve.js
    ├── service_cert.pem
    ├── set_js_app_proposal.json
    ├── set_js_app_vote_for.json
    ├── user0_cert.pem
    ├── user0_privk.pem
    └── validate.js
```